examples/Swift SDK/Swift SDK/AuthorizationCodeGrantExampleViewController.swift

```diff
@@ -168,30 +166,26 @@ class AuthorizationCodeGrantExampleViewController: AuthorizationBaseViewControll
                 self.carLabel.text = "\(make) \(model)\n(\(licensePlate)"
                 
                 // Asynchronously fetch images
-                if let driverUrl = URL(string: driverImage) {
-                    URLSession.shared.dataTask(with: driverUrl) {
-                        (data, response, error) in
-                        DispatchQueue.main.async {
-                            guard let data = data else {
-                                return
-                            }
-                            
-                            self.driverImageView.image = UIImage(data: data)
+                URLSession.shared.dataTask(with: driverImage) {
```

Comment:
Do we have a singleton for our HTTP session?

Reply:
Yes, it's a singleton for our application provided by the UIKit framework.